cmd/triggers/generate.go

```diff
@@ -171,8 +171,7 @@ func getTriggerPaths(sdkCLIConfig *hooks.SDKCLIConfig) []string {
 
 // getFullyQualifiedTriggerFilePaths returns an array of file paths that
 // are validated
-func getFullyQualifiedTriggerFilePaths(clients *shared.ClientFactory, triggerPaths []string) ([]string, error) {
-	ctx := context.Background()
```

Comment:
note: This is 1st key change, where we remove `ctx := context.Background()` and replace wit with a `ctx context.Context` function argument.